files/opt/dash-button/scripts/script.js

```diff
@@ -0,0 +1,67 @@
+const button = require('node-dash-button');
+const https = require('https');
+const request = require('request');
+var fs = require('node-fs');
+
+// Load config.json
+const configJSON = fs.readFileSync(process.cwd() +"/scripts/config.json");
+const config = JSON.parse(configJSON);
+const buttonConfig = config.buttons;
+const providerConfig = config.providerConfig;
+
+// Add handler for dash-buttons
+buttonConfig.forEach(function(value) {
+    console.log('Configuring button {' + value.mac + '} for product {' + value.providerConfig.productName + '} with provider ' + value.provider);
```

Comment:
Es ist hier nicht gut auf `value.providerConfig.productName` zuzugreifen, weil ja evtl. Buttons kofniguriert werden, die überhaupt nicht auf Bring konfiguriert sind.

Besser wäre also z.B. `Configuring Button XXXX for provider YYYY ([Config-Values]).`